swim/node.go

```diff
@@ -510,3 +510,7 @@ func (n *Node) pingNextMember() {
 		"numErrors": len(errs),
 	}).Warn("ping request inconclusive due to errors")
 }
+
+func (n *Node) GetMembers() []string {
```

Comment:
Suggest renaming the function as GetActiveMembers. Same comment for the ringpop method. other than that, lgtm